src/js/routes/Ballot/Ballot.jsx

```diff
@@ -349,7 +354,7 @@ export default class Ballot extends Component {
             </h1>
           </OverlayTrigger> :
           null }
-        <EditAddress address={voter_address} _toggleSelectAddressModal={this._toggleSelectAddressModal} />
+        <EditAddress address={voter_address} isNormalized={is_normalized} _toggleSelectAddressModal={this._toggleSelectAddressModal} />
```

Comment:
Hi Edward, could you please refactor EditAddress to take in the address={address_object}, so you don't need to calculate "isNormalized" external to the EditAddress component? (I am going to approve this pull request as-is, so there is no rush.)